test/ventilation_test.c

```diff
@@ -29,16 +32,60 @@ int startVentilation_afterOneInspiration_openInspiratoryValve() {
 
   //reinitialize the system to be sure that the valve wasn't previously open
   close_inspiratory_valve();
+  reset_to_inspiration_start();
   
   start_inspiration();
 
   int inspiratory_valve_status = get_inspiratory_valve_status();
   assertTrue(inspiratory_valve_status == VALVE_OPEN);
 }
 
+int startVentilation_afterOneInspiration_setsVentilationPhaseToInspiration() {
+  //initialize the function get_current_time
+  int* times = (int*) malloc(1*sizeof(int)); 
+  times[0] = 260;
```

Comment:
The numbers in these tests seem to be dependent on the hardcoded value `breath_cycle_duration = 125` in `ventilation.c`.  

* Why is this the case? Should the breath cycle duration not be computed and pulled in from else where?
* It is bad to hardcode the `125` and link to the values in this unit test, it makes maintaining the unit tests hardware in the future.

Reply:
done